frontend/src/components/management-data/DocumentUnitManagementData.vue

```diff
@@ -76,16 +153,63 @@ const loadHistory = async () => {
           </dd>
         </div>
       </dl>
-    </div>
-    <div class="flex flex-col gap-24 bg-white p-24">
-      <TitleElement
-        >Dokumentationseinheit "{{ documentUnit?.documentNumber }}"
-        löschen</TitleElement
-      >
-      <DocumentUnitDeleteButton
-        :document-number="documentUnit?.documentNumber!"
-        :uuid="documentUnit?.uuid!"
-      />
+      <div v-if="assignDocOfficeResponseError">
+        <InfoModal
+          data-testid="assignDocOfficeErrorModal"
+          :description="assignDocOfficeResponseError.description"
+          :title="assignDocOfficeResponseError.title"
+        />
+      </div>
+      <dl>
+        <div class="flex gap-24 px-0">
+          <dt
+            class="ris-body1-bold shrink-0 grow-0 basis-160"
+            data-testid="assign-documentation-office-title"
+          >
+            Zuweisen
+          </dt>
+          <dd class="ris-body2-regular w-full gap-32">
+            <div class="flex flex-wrap gap-8">
+              <div class="w-[320px]">
```

Comment:
Arbitrary Values vermeiden. -> Tailwind hat eine[default width scale](https://tailwindcss.com/docs/width) das bis w-96 (~384px) geht. Falls wirklich eine feste Breite gebraucht wird (was ich bei inputs immer vermeiden würde), dann benutze am besten diese default Werte. Wenn es ein custom Wert sein muss, füge es dem @theme in der global.css als neues tailwind token hinzu, kann dir gerne zeigen wie das geht, wenn du es brauchst.

Reply:
Ich würde die Breite hier einfach rausnehmen, oder der combobox eine class="min-w-96" mitgeben im Notfall